Core/clim-core/menu.lisp

```diff
@@ -50,6 +50,11 @@
       (dispatch-repaint button (sheet-region button))
       (stream-force-output button))))
 
+(defgeneric toggle-arm-branch (button))
+
+(defmethod toggle-arm-branch ((button menu-button-pane))
+  (arm-menu button))
+
```

Comment:
please rename arm-menu function to toggle-arm-branch and merge their behaviors. also write a comment above defgeneric form which describe how we achieve desired behavior.

Reply:
Can't merge arm-menu and toggle-arm-branch.
arm-menu when menu is already armed. toggle-arm-branch is solely for pointer-button-press-event.

I can remove toggle-arm-branch by overriding handle-event for menu-button-submenu-pane.